### ISSUE TITLE
products/includes/header.template.html

### ISSUE THREAD

**Issue by Redshift-Web-Design** [NetoECommerce/Skeletal#69]
Pinterest share link is missing the 'media=' declaration which means pinterest shares won't display the image which is kind of the point of pinterest :-)
The pinterest hyperlink should be.

```
"//www.pinterest.com/pin/create/button/?url=[%url_encode%][@url@][%/url_encode%]&amp;media=[%asset_url type:'product' id:'[@SKU@]' thumb:'full' check_parent:'y'%][%END asset_url%]&amp;description=[%url_encode%][@name@][%/url_encode%]"
```

**Comment by brandonsheppard** [comment]
You might be interested in this, @dalerodgie

**Comment by dalerodgie** [comment]
Pinterest requires the full web address for the image. Use this instead:

~~~
//www.pinterest.com/pin/create/button/?url=[%url_encode%][@url@][%/url_encode%]&amp;media=[%url_encode%][@config:homeurl@][%asset_url type:'product' id:'[@SKU@]' thumb:'full' check_parent:'y'%][%END asset_url%][%/url_encode%]&amp;description=[%url_encode%][@name@][%/url_encode%]
~~~

changed:

~~~
media=[%url_encode%][@config:homeurl@][%asset_url type:'product' id:'[@SKU@]' thumb:'full' check_parent:'y'%][%END asset_url%][%/url_encode%]
~~~

**Comment by Redshift-Web-Design** [comment]
Yeah I ran into that issue when updating the content pages. Seemed to work on product pages as it was though. Will update. Is there a list of [@configs@] documented somewhere?

Thanks guys.

**Comment by brandonsheppard** [comment]
Fixed! c9b7438bce027c7c13caa92db26ecff957daea6a

@Redshift-Web-Design: There is not really a comprehensive database of these but that's on the short-term to-do list that's for sure. 

That said, if you hit-up the advanced configs page you can see them all, and their title or name is the config.

But be careful—**with great power comes great responsibility**

![](http://i.giphy.com/HOnL1e8Jyyqpq.gif)

**Comment by Redshift-Web-Design** [comment]
Hahaha, that could be the single best animated gif I have seen. Raddddd… Just brightened up all the staffs day here.

 

Thanks mate.

 

--
